src/adagrams.js

```diff
@@ -1,15 +1,104 @@
+import Chance from "chance";
```

Comment:
Love that you researched and implemented a javascript library!

I'm also curious how you could have done this with vanilla JS if you didn't have access to a library like this. 🤔